src/main/resources/org/jenkinsci/plugins/ownership/model/workflow/OwnershipGlobalVariable/help.jelly

```diff
@@ -0,0 +1,69 @@
+<?jelly escape-by-default='true'?>
+<div>
+ Provides ownership info for different item types.
+ <p/>
+ Currently the macro supports retrieval of ownership info for jobs and nodes only.
+ It can be done via the nested variables:
+<dl>
+  <dt>ownership.job</dt>
+  <dd>
+    Job ownership info.
+    This info may be inherited from parent folders.
+  </dd>
+  <dt>ownership.node</dt>
+  <dd>
+    Node ownership info. 
+    The expression must be used within the <code>node()</code> Pipeline block.
```

Comment:
node(), because we use brackets to specify label expressions. "{}" is defined as "Pipeline block" here